### PR TITLE
Containerization

### PR DESCRIPTION
**Pull request by dziyanabahdanava**

### CODE REVIEW

File: src/main/resources/application-local.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+caloriescalculator:
+  weightFactor: 9.99
+  heightFactor: 6.25
+  ageFactor: -4.92
+  deduction: -161
+management:
+  endpoints.web.exposure.include: health,info,refresh
+clients:
+  user-service:
+    url: http://user-service:8080
```
Comment (TsybulskyStepan) [inline src/main/resources/application-local.yml]: do you use it?

File: src/test/java/com/epam/ms/contract/UserServiceContractTest.java [hunk 2]
```diff
@@ -13,82 +13,76 @@
...
-        Map<String, String> headers = new HashMap<>();
-        headers.put("Content-Type", "application/json");
-        UserProfile defaultProfile = createDefaultProfile();
-        String defaultJSON = "";
-        try{
-            defaultJSON = mapper.writeValueAsString(defaultProfile);
-        } catch (JsonProcessingException e) {
-
-        }
+//    @Pact(consumer = "nutritionist_service", provider = "user_service")
```
Comment (TsybulskyStepan) [inline src/test/java/com/epam/ms/contract/UserServiceContractTest.java]: if you don't need it , then delete :)

**Review by TsybulskyStepan — APPROVED** [review]

File: src/test/java/com/epam/ms/contract/UserServiceContractTest.java (hunk 2 above)
Comment (dziyanabahdanava) [reply src/test/java/com/epam/ms/contract/UserServiceContractTest.java]: Thank you, fixed :)

File: src/main/resources/application-local.yml (hunk 1 above)
Comment (dziyanabahdanava) [reply src/main/resources/application-local.yml]: Thank you, deleted

**Pull request merged**